pkg/apis/storage/types.go

```diff
@@ -424,3 +436,67 @@ type CSINodeList struct {
 	// items is the list of CSINode
 	Items []CSINode
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CSIStorageCapacity stores the result of one CSI GetCapacity call for one
+// driver, one topology segment, and the parameters of one storage class.
```

Comment:
I could stand to see a bit more docs here.  E.g.

```
For a given StorageClass, this describes the available capacity in a particular topology.  This can be used when considering where to instantiate new PersistentVolumes.  For example this can express things like:
   - StorageClass "standard" has "1234 GiB" available in "topology.kubernetes.io/zone=us-east1" 
   - StorageClass "localssd" has "10 GiB" available in "kubernetes.io/hostname=knode-abc123"
```

This would be a good place to document the semantics of edge cases like "no object for zone us-east2" or "it has an object but capacity is unset", so potential consumers of this API understand what it is for and how to use it properly.

Reply:
Done.